### PR TITLE
Add iPhone XR, iPhone XS iPhone XS Max device models to a getModels function

### PR DESCRIPTION
**Pull request by tigranbalayan**
## Description

Fixed issue #493

### CODE REVIEW

File: ios/RNDeviceInfo/RNDeviceInfo.m [hunk 1]
```diff
@@ -103,6 +103,9 @@ - (NSString*) deviceName
                               @"iPhone9,4" :@"iPhone 7 Plus",   // (model A1784 | Global)
                               @"iPhone10,3":@"iPhone X",        // (model A1865, A1902)
                               @"iPhone10,6":@"iPhone X",        // (model A1901)
+                              @"iPhone11,8":@"iPhone XR",       // (model A1882, A1719)
+                              @"iPhone11,2":@"iPhone XS",       //
+                              @"iPhone11,4":@"iPhone XS Max",   //
```
[the remark this answers is not in the document]
Comment (psegalen) [reply ios/RNDeviceInfo/RNDeviceInfo.m]: Same information here: https://www.theiphonewiki.com/wiki/IPhone_XS_Max

[the remark this answers is not in the document]
Comment (tigranbalayan) [reply ios/RNDeviceInfo/RNDeviceInfo.m]: When we can have a new version with new iPhones in Your opinion?

[the remark this answers is not in the document]
Comment (machour) [reply ios/RNDeviceInfo/RNDeviceInfo.m]: Hi, I'm waiting until this PR is finished. Could you take in account @psegalen and @benadamstyles ?

[the remark this answers is not in the document]
Comment (duro) [reply ios/RNDeviceInfo/RNDeviceInfo.m]: @machour Can someone else just add that one line and get this shipped? These phones hit the shelves today, and I'd like to tighten this up ASAP.

[the remark this answers is not in the document]
Comment (duro) [reply ios/RNDeviceInfo/RNDeviceInfo.m]: @machour I have submitted a new PR that combines all of these

**Comment by machour** [comment]
Replaced by #499.

**Pull request closed without merging**